orbotservice/src/main/assets/fronts

```diff
@@ -1,13 +1,3 @@
-snowflake-target https://1098762253.rsc.cdn77.org/
-snowflake-front www.cdn77.com,www.phpmyadmin.net
-snowflake-stun stun:stun.antisip.com:3478,stun:stun.epygi.com:3478,stun:stun.uls.co.za:3478,stun:stun.voipgate.com:3478,stun:stun.mixvoip.com:3478,stun:stun.nextcloud.com:3478,stun:stun.bethesda.net:3478,stun:stun.nextcloud.com:443
 snowflake-target-direct https://snowflake-broker.torproject.net/
```

Comment:
Keeping these variables in one location was easier to maintain. I've noticed some of these are now moved into various locations.